src/main/java/com/oltpbenchmark/DBWorkload.java

```diff
@@ -475,35 +207,86 @@ public static void main(String[] args) throws Exception {
         }
     }
 
-    private static Options buildOptions(XMLConfiguration pluginConfig) {
+    private static List<Phase> getPhases(Workload workload) {
```

Comment:
@lmwnshn here is the new method where i attempted to convert the phase logic and the interpret the new enum values